HTTPSConnectionPool(host='data.alpaca.markets', port=443): Max retries exceeded with url: /v1beta3/crypto/us/bars?start=2023-09-04T00%3A00%3A00%2B00%3A00&end=2024-02-01T01%3A00%3A57.794909%2B00%3A00&timeframe=5Min&symbols=ETH%2FUSD (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x0000027576E86F50>, 'Connection to data.alpaca.markets timed out. (connect timeout=None)'))
